django/contrib/admindocs/templates/admin_doc/bookmarklets.html

```diff
@@ -12,7 +12,7 @@
 
 {% block content %}
 
-<p class="help">{% blocktranslate trimmed %}
+<p>{% blocktranslate trimmed %}
```

Comment:
I think we should change this to `quiet` to keep the previous color `color: var(--body-quiet-color)`:
```suggestion
<p class="quiet">{% blocktranslate trimmed %}
```

Reply:
@specialorange Can you take a look? :point_up:

Reply:
done, now to try to figure out how to PR the new files instead